### PR TITLE
develop/hardware: Add IOT-GATE-iMX8 uart mode instructions

### PR DESCRIPTION
**Pull request by acostach**
Change-type: patch
Signed-off-by: Alexandru Costache <alexandru@balena.io>

uart_mode on the iot-gate-imx8 will be managed by the supervisor once https://github.com/balena-os/balena-supervisor/issues/1948 is implemented

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/learn/develop/hardware/i2c-and-spi.md [hunk 1]
```diff
@@ -296,6 +298,15 @@ sleep 5s && ifconfig usb0 down
...
 After this you should be able to easily use your Intel Edison in USB host mode.
 
+## IOT-GATE-iMX8
+
+### Serial ports
+The IOT-GATE-iMX8 has at least one serial port that can be configured as either RS232 or RS485. To select the rs232 operating mode, in the host operating system, add the following line to `/mnt/boot/extra_uEnv.txt`:
+```Bash
+uart_mode=rs232
+```
+Similarly, rs485 can be selected in the environment file. The device should be rebooted after setting the uart operating mode the for the changes to take effect.
```
Comment (vipulgupta2048) [inline pages/learn/develop/hardware/i2c-and-spi.md:308]:
```suggestion
### Serial ports

The IOT-GATE-iMX8 has at least one serial port that can be configured as either RS232 or RS485. To select the rs232 operating mode in the host operating system, add the following line to `/mnt/boot/extra_uEnv.txt`:

```bash
uart_mode=rs232
```

Similarly, rs485 can be selected in the environment file as well. The device should be rebooted after setting the UART operating mode for the changes to take effect.
```

**Review by vipulgupta2048 — COMMENTED** [review]
Looking good

File: pages/learn/develop/hardware/i2c-and-spi.md [hunk 2]
```diff
@@ -296,6 +298,15 @@ sleep 5s && ifconfig usb0 down
 
 After this you should be able to easily use your Intel Edison in USB host mode.
 
+## IOT-GATE-iMX8
+
+### Serial ports
+The IOT-GATE-iMX8 has at least one serial port that can be configured as either RS232 or RS485. To select the rs232 operating mode, in the host operating system, add the following line to `/mnt/boot/extra_uEnv.txt`:
```
Comment (vipulgupta2048) [inline pages/learn/develop/hardware/i2c-and-spi.md:304]:
```suggestion
### Serial ports

The IOT-GATE-iMX8 has at least one serial port that can be configured as either RS232 or RS485. To select the rs232 operating mode in the host operating system, add the following line to `/mnt/boot/extra_uEnv.txt`:

```

Comment (acostach) [reply pages/learn/develop/hardware/i2c-and-spi.md]: Thanks Vipul, solved

**Comment by acostach** [comment]
@balena-ci I self-certify!

**Pull request merged**